l argument was formerly ignored by binutils, but since 2.36 is used
for something other than what was intended here (tmp files location).
https://bugs.gentoo.org/782514
--- a/Imake.tmpl
+++ b/Imake.tmpl
@@ -1083,3 +1083,3 @@
 #else
-#define ArCmd ArCmdBase clq
+#define ArCmd ArCmdBase cq
 #endif
@@ -1090,3 +1090,3 @@
 #else
-#define ArAddCmd ArCmdBase rul
+#define ArAddCmd ArCmdBase ru
 #endif
@@ -1097,3 +1097,3 @@
 #else
-#define ArExtCmd ArCmdBase xl
+#define ArExtCmd ArCmdBase x
 #endif
